src/Server.php

```diff
@@ -22,24 +24,21 @@ public function __construct($host, $port)
     public function run($force = false)
     {
         if ($force) {
-                $this->killProgramsOnDreddPort();
+            $this->killProgramsOnDreddPort();
         }
 
         $socket = sprintf('tcp://%s:%s', $this->host, $this->port);
         $server = stream_socket_server($socket, $errno, $errorMessage);
 
         if ($server === false) {
-
             throw new UnexpectedValueException("Server could not bind to socket: $errorMessage");
         }
 
         $buffer = "";
 
-        for (;;) {
+        while ($connection = @stream_socket_accept($server)) {
```

Comment:
Remove the `@` this means the user won't see any output when this fails and those warnings are helpful.